.eslintrc

```diff
@@ -922,6 +926,7 @@
   "plugins": [
     "import",
     "react",
+    "react-hooks",
```

Comment:
So my basic thinking is:

* Hooks are good and class components are bad (more on that below)
* We should therefore not introduce any new class components, and when we need the features that require either hooks or class components in a new component, we should use hooks
* And soon we should probably transition fully to hooks even though the React folks say not to go all crazy rewriting components for hooks just yet
* So, with that in mind, this is the first time since hooks became a thing that we have a new component that needs to use hooks. So, this would be the exact time to "start supporting hooks" (I don't see much argument for, say, introducing the eslint plugin when there are no actual hooks to lint)

Reply:
Since writing this I'm now developing almost entirely in hooks, I think I'll write up some thoughts on that on a higher-up comment.